pom.xml

```diff
@@ -202,6 +202,9 @@
     <arrow.version>0.15.1</arrow.version>
     <!-- org.fusesource.leveldbjni will be used except on arm64 platform. -->
     <leveldbjni.group>org.fusesource.leveldbjni</leveldbjni.group>
+    
+    <!-- ozone version -->
+    <ozone.version>0.5.0-beta</ozone.version>
```

Comment:
Thank you for making a PR, but we usually don't add `-beta` dependency.

cc @dbtsai and @gatorsmile

Reply:
Ozone will be GA next release. Discussion is happening in the community around this topic.

Ozone has been scale tested with 1 billion objects and also it is tested with TPCSDS Hive and Spark.

Reply:
What is the next `Ozone will be GA next release`? Could you close this and reopen after that with the official version?

Reply:
I marked this PR as `[WIP]`. Please remove that when the PR is ready.

Reply:
Ozone's next release is 0.6.0. 
Mostly it will be released before August 2020.